test/explorer/data_frame_test.exs

```diff
@@ -685,6 +685,27 @@ defmodule Explorer.DataFrameTest do
              }
     end
 
+    test "adds some columns with select functions" do
+      a = Series.from_list([true, false, true])
+      b = Series.from_list([3, 4, 2])
+      c = Series.from_list([6, 2, 1])
+      df = DF.new(a: a, b: b, c: c)
+
+      df1 =
+        DF.mutate_with(df, fn ldf ->
+          [
+            select1: Series.select(ldf["a"], ldf["b"], ldf["c"])
+          ]
+        end)
+
+      assert DF.to_columns(df1, atom_keys: true) == %{
+               a: [true, false, true],
+               b: [3, 4, 2],
+               c: [6, 2, 1],
+               select1: [3, 2, 2]
+             }
+    end
```

Comment:
Cool! Would you mind to add a test for the `Explorer.Series.select/3` function as well? The idea is to also have a test for the Series module.